Add layer field to event (#8881)
apache/skywalking: docs/en/changes/changes.md

* Fix OAP Self-Observability metric `GC Time` calculation.
* Set `SW_QUERY_MAX_QUERY_COMPLEXITY` default value to `1000`
* Webapp module (for UI) enabled compression.

> * [Breaking Change] Add layer field to event, report an event without layer will not be allowed.

Reviewer: ```suggestion
* [Breaking Change] Add layer field to event, report an event without layer is be allowed.
```
 Future tense in the change log is strange.

Author: make sense to me
